### PR TITLE
[17.0][MIG] fieldservice

### PR DESCRIPTION
**Pull request by brian10048**
Ready for functional testing on runboat.

### CODE REVIEW

**Comment by SMaciasOSI** [comment]
@brian10048 #1141 Has been merged. Please fix the pre-commit issues with this PR so we can test and review

**Comment by brian10048** [comment]
@SMaciasOSI All set for testing and review

File: fieldservice/views/fsm_team.xml [hunk 1]
```diff
@@ -0,0 +1,273 @@
+<odoo>
+    <record id="fsm_order_action_from_dashboard" model="ir.actions.act_window">
+        <field name="name">Orders</field>
+        <field name="res_model">fsm.order</field>
+        <field name="view_mode">kanban,tree,form,calendar</field>
+        <field name="context">{'default_team_id': id}</field>
```
Comment (SMaciasOSI) [inline fieldservice/views/fsm_team.xml]: @brian10048 This line is giving me an error when I try to click the "To do" button in the kanban view 
Captura desde 2023-11-27 12-20-37

**Comment by SMaciasOSI** [comment]
@brian10048 I have tested your changes on runboat, the JS error has disappeared. Now, the precommit and tests checks are not being triggered for some reason. Any idea why?

File: fieldservice/views/fsm_team.xml (hunk 1 above)
Comment (SMaciasOSI) [reply fieldservice/views/fsm_team.xml]: This has been solved

**Comment by brian10048** [comment]
@SMaciasOSI I think it was an issue with Github.  I bumped it just now and it is running the checks

**Comment by SMaciasOSI** [comment]
@brian10048 All seems good. As soon as the codecov checks pass, I'll approve this PR.

**Comment by brian10048** [comment]
@SMaciasOSI I think I'm out of hours budget to work anymore on this to increase the coverage above 98.8% :laughing: 

There are only a few lines which are partially covered, with no lines missed

**Review by max3903 — APPROVED** [review]

**Review by SMaciasOSI — APPROVED** [review]

**Pull request merged**